src/app/components/PortraitVideoModal/index.styles.tsx

```diff
@@ -0,0 +1,81 @@
+import { css, Theme } from '@emotion/react';
+
+const styles = {
+  modalWrapper: (theme: Theme) =>
+    css({
+      position: 'fixed',
+      top: 0,
+      left: 0,
+      zIndex: 9999,
+      width: '100vw',
+      height: '100vh',
+      backgroundColor: 'rgba(0, 0, 0, 0.9)',
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      padding: '1rem',
+    }),
+
+  modalInner: css({
+    width: '100%',
+    maxWidth: '480px',
+    position: 'relative', // anchor the nav buttons
```

Comment:
These will need to be changed to suit the new UX update which seems to have vertical swiping rather than LTR/RTL. So there are two pairs of chevrons for views which will mean an `up` chevron goes `right` and a `down` chevron goes `left` across the carousel.